src/Adapter/Module/Repository/ModuleRepository.php

```diff
@@ -128,6 +133,30 @@ public function getActiveModules(): array
         return $activeModules;
     }
 
+    /**
+     * Return present modules (even if those not installed in DB).
+     *
+     * @return array
+     */
+    public function getPresentModules(): array
+    {
+        if (!defined('_DB_PREFIX_')) {
+            return []; // getActiveModules() can be called during install BEFORE the database configuration has been defined
```

Comment:
```suggestion
            return []; // getPresentModules() can't be called during install BEFORE the database configuration has been defined
```

Reply:
Nope, that's the reason why there is a fallback because it can happen and it would fail
But the method name is wrong indeed

Reply:
Oh but it made me realize this method makes no call to the DB, so I should remove this check